### ISSUE TITLE
learning rate adjustment

### ISSUE THREAD

**Issue by Ivamcoder** [bearpaw/pytorch-pose#84]
Hi @bearpaw 
the plot from log file shows that network is trained for `220` epoch and we have adjustment on `150th` epoch,
but in code , I did not find any adjustment , would you please share adjustment for learning rate to get your results?

**Comment by bearpaw** [comment]
You can use arguments to set the epoch, lr, and lr decay strategy
- for training epoch, see https://github.com/bearpaw/pytorch-pose/blob/master/example/main.py#L383
- for lr decay, see: https://github.com/bearpaw/pytorch-pose/blob/master/example/main.py#L397
